packages/pci-sdk-iframe/src/pages/app/components/Debugger.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { PropsWithChildren, useEffect, useState } from 'react';
+
+type IDebuggerProps = any;
+
+export default function Debugger(props: IDebuggerProps) {
```

Comment:
We don't seem to be using any `props`, hence I think we can remove `IDebuggerProps`? ☕️

Reply:
I think he is using it to print the body in the `<Response />` component.